docs/config/README.md

```diff
@@ -0,0 +1,191 @@
+# Minio Server Configuration Guide [![Slack](https://slack.minio.io/slack?type=svg)](https://slack.minio.io) [![Go Report Card](https://goreportcard.com/badge/minio/minio)](https://goreportcard.com/report/minio/minio) [![Docker Pulls](https://img.shields.io/docker/pulls/minio/minio.svg?maxAge=604800)](https://hub.docker.com/r/minio/minio/) [![codecov](https://codecov.io/gh/minio/minio/branch/master/graph/badge.svg)](https://codecov.io/gh/minio/minio)
+
+In this document we will walk through the Minio Server configuration files. Minio Server stores all the necessary configuration parameters in `${HOME}/.minio/config.json` , following fields provide more detailed understanding of each of these fields and how we can configure them.
+
+## Minio Server configuration directory
+
+Minio Server uses `${HOME}/.minio` as the default configuration directory.  If this directory doesn't exist it is created by Minio Server upon a fresh start. Minio Server can also be configured to use a custom configuration directory by simply specifying `--config-dir` command line option.
+
+```sh
+minio server --config-dir /etc/minio
+```
+
+## Minio Server config directory
+
+Following is the directory structure for default Minio Server configuration.
+
+```sh
+$ tree ~/.minio
+/home/miniouser/.minio
+├── certs
+│   └── CAs
+└── config.json
+
+2 directories, 1 file
+```
+
+Following is the directory structure for Minio Server configured with TLS.
+
+```sh
+$ tree ~/.minio
+/home/miniouser/.minio
+├── certs
+│   ├── CAs
+│   ├── private.key
+│   └── public.crt
+└── config.json
+
+2 directories, 3 files
+```
+### Certificate directory
+``certs`` directory stores key & cert information, which are needed to run Minio in ``HTTPS``. You can read more on [How to secure access to Minio server with TLS](http://docs.minio.io/docs/how-to-secure-access-to-minio-server-with-tls)
+
+
+### Configuration file
```

Comment:
May be make it 

## Minio Server configuration file

and change below sections accordingly..